### ISSUE TITLE
Warnings in Logs

### ISSUE THREAD

**Issue by Lucas-M-G** [muety/wakapi#153]
Are the following warnings I found in the logs relevant?

[WARN ] did not insert key 'latest_total_time', maybe just updated?
[WARN ] did not insert key 'latest_total_users', maybe just updated?

I use wakapi in a docker container with MySQL (Maria DB) and behind an nginx reverse proxy.

**Comment by muety** [comment]
Thanks for bringing this up. It's actually a known issue and due to the fact that the upstream WakaTime CLI won't use your custom API endpoint for retrieving that stats, but instead attempts to query `api.wakatime.com`. Here is some explanation:

[This](https://github.com/wakatime/vscode-wakatime/blob/c3e3ec510019441888c28a0c1726ce4456414a7c/src/wakatime.ts#L415) is where the VSCode plugin calls the CLI to fetch the stats. And [this](https://github.com/wakatime/wakatime/blob/e8deb156f1c2d26e5cf874da97f7b4354b3f5d20/wakatime/api.py#L173) is where the CLI does the actual API Call.

As can be seen, the endpoint is hard-coded there, so the request will never reach Wakapi.

I'll file an issue in the upstream repo, maybe they can get this fixed.

**Comment by muety** [comment]
> Are the following warnings I found in the logs relevant?

No, they can be ignored. Actually, I'll just remove them with the next release, as they don't provide any useful information anyway and might only confuse people.

**Comment by Lucas-M-G** [comment]
Thanks for your reply and the awesome software.

**Comment by kondr1** [comment]
@muety but now wakatime use another cli. 
[Here](https://github.com/wakatime/vscode-wakatime/blob/master/src/wakatime.ts#L435) is a plugin that uses golang binary cli. And [here](https://github.com/wakatime/wakatime-cli/blob/develop/pkg/api/summary.go#L18) is how this cli works. 
Endpoint not hardcoded already =)

**Comment by muety** [comment]
This issue will be resolved with #244 (hopefully soon :wink:).

**Comment by gaocegege** [comment]
It works for me, can we close the issue?
